src/pages/about-us.tsx

```diff
@@ -23,36 +23,30 @@ const AboutUs = (props: PageProps<AboutUsQueryProps>) => {
       />
       <AboutUsPage
         heroImageData={props.data.hero}
-        team={props.data.allContentfulTeamMember.nodes}
+        images={props.data.allContentfulGalleryImage.nodes}
       />
     </>
   );
 };
 
 export default AboutUs;
+
 export const AboutUsPageQuery = graphql`
   query ($language: String!) {
-    hero: file(relativePath: { eq: "office/sy-office-02.jpg" }) {
+    hero: file(relativePath: { eq: "office/sy-office-05.jpg" }) {
       childImageSharp {
         gatsbyImageData(layout: FULL_WIDTH)
       }
     }
 
-    allContentfulTeamMember(filter: { node_locale: { eq: "en" } }) {
+    allContentfulGalleryImage(
+      filter: { node_locale: { eq: "en" } }
+      sort: { index: ASC }
+    ) {
       nodes {
         id
-        name
         image {
-          gatsbyImageData(
-            width: 400
-            aspectRatio: 1
-            layout: CONSTRAINED
-            placeholder: DOMINANT_COLOR
-            formats: [AUTO, WEBP, AVIF]
-            cropFocus: FACE
-            resizingBehavior: FILL
-            jpegProgressive: false
-          )
+          gatsbyImageData(resizingBehavior: FILL, jpegProgressive: false)
```

Comment:
I see many faces do show a cut off which comes from the design input. Nothing to blame technical-wise. But what about the formats that you have omitted and the placeholder? How did you decide about that?